e2e-tests/cypress/integration/customizer/layout/homepage-settings.spec.ts

```diff
@@ -0,0 +1,28 @@
+describe('Homepage-settings', function () {
+	beforeEach(function () {
+		cy.loginWithRequest('/wp-admin/customize.php');
+		cy.get('#accordion-panel-neve_layout > .accordion-section-title').click();
+		cy.get('#accordion-section-static_front_page > .accordion-section-title').click();
+		cy.get('#_customize-input-show_on_front-radio-page').click();
+	});
+
+	it('Sets up a page to be the homepage', function () {
+		/** TODO */
+
+		cy.get('#_customize-input-page_on_front').select('Sample Page');
+		cy.get('#save').click({ force: true });
+		cy.visit('/');
+		cy.get('h1').should('contain.text', 'Sample Page');
+	});
+
+	it.only('Sets up a page to be the posts page', function () {
```

Comment:
the .only is used for running only one test from the suite. Do we want to run only this test or the entire suite?